Add standard vesting schedule and ISO/NSO info (#2197)
sourcegraph/about: handbook/people-ops/compensation.md

We are an early-stage technology company, and you can have a huge impact on our success. We want to make sure that you have a financial stake in the success and that your contributions are rewarded.

> Equity grants vest monthly with a 1 year cliff.

Reviewer: ```suggestion
By default, all equity grants vest monthly with a 1-year cliff.
```

> Teammates within the US receive incentive stock options (ISOs) and teammates outside of the US receive non-qualified stock option (NSOs).

Reviewer: ```suggestion
Teammates classified as employees receive incentive stock options (ISOs) to the maximum degree possible, while teammates classified as contractors receive non-qualified stock option (NSOs).
```

We are an early-stage technology company, and you can have a huge impact on our success. We want to make sure that you have a financial stake in the success and that your contributions are rewarded.

Equity grants vest monthly with a 1 year cliff.

> TODO(@sourcegraph/people-ops): add more detail here about the kind of options and how to understand their value.

Author: ```suggestion
Teammates within the US receive incentive stock options (ISOs) and teammates outside of the US receive non-qualified stock option (NSOs).

TODO(@sourcegraph/people-ops): add more detail here about the kind of options and how to understand their value.
```
